frontend/src/frontend/code_generator/user_callgraph.py

```diff
@@ -143,10 +143,60 @@ def format_branch(self, branch: blocks.Branch) -> str:
             return branch_formatters[branch.branch_type](branch)
         raise ValueError(f'Unknown branch type: {branch.branch_type}')
 
-    def _format_branch_indirect_call(self, branch: blocks.Branch) -> str:
+    def _format_branch_indirect_call(self,
+                                     branch: blocks.Branch,
+                                     uuid: int = None) -> str:
```

Comment:
Can we clarify what the uuid is uniquely representing? I think it's representing all the elements needed to perform this indirect branch.

Reply:
It's a unique suffix used to differentiate multiple branches in a single function and to avoid any overlapping variables provided in user defined instructions.
Updated with docstring.